### DISCUSSION TITLE
A little css fix

### DISCUSSION THREAD

**Discussion by liucoj** [open-webui/open-webui#1649] [General]
Hi there,
For a bit of time in the main screen, above where you choose the model, there are two black blocks that prevent visualization. Moreover, even the left button to create a new chat has incorrect formatting text.
I use OpenWebui on different Macs (M2 Air, MacMini M1 with 24", MacBook Pro M2 15") with same issue.
 
<img width="1582" alt="Screenshot 2024-04-21 alle 11 17 07" src="https://github.com/open-webui/open-webui/assets/14237462/2a24d7f0-37fe-4216-b96d-87749e7caddc">

**Comment by tjbck** [comment]
Please use `npm` to install and run/build. Running `npm install` again should fix the issue!

**Reply by liucoj** [reply]
thank you! running npm install / npm run build solved this issue.. 
I used to bun install / bun run build
